src/spryker/application/skeleton/phpstan.neon

```diff
@@ -6,6 +6,7 @@ parameters:
     excludes_analyse:
         - %rootDir%/../../../src/Generated/*
         - %rootDir%/../../../src/Orm/*
+        - %rootDir%/../../../src/Pyz/Zed/DataImport/Business/*
```

Comment:
This module has some phpstan issues, which I couldn't fix easily. That is why I have ignored it for now.